Library/Homebrew/rubocops/dependency_order.rb

```diff
@@ -75,7 +75,7 @@ def sort_conditional_dependencies!(ordered)
             ordered.each_with_index do |dep, pos|
               idx = pos+1
               match_nodes = build_with_dependency_name(dep)
-              next if !match_nodes || match_nodes.empty?
+              next if match_nodes.blank?
```

Comment:
As noted above, `blank?` is been available (and used) within our custom rubocops already, so i've removed the exclusions, pending refactor